docs/getting-started/quickstart-programming.md

```diff
@@ -3,24 +3,11 @@ sidebar_position: 2
 title: Programming Quickstart
 ---
 
-Let's learn how to build applications with [DBOS Transact](https://github.com/dbos-inc/dbos-ts), the open-source TypeScript framework for DBOS.
-In this tutorial, we will modify the example application from our [quickstart](./quickstart.md) to reliably send a greeting note to your friends.
-Along the way, we'll introduce you to core DBOS concepts and show how you can easily build a transactional and reliable application.
-First, you'll learn to create HTTP endpoints to serve requests.
-Then, you'll learn how to interact with a database and make third-party API calls from your application.
-Finally, you'll compose these steps in reliable workflows.
+Let's learn how to build applications with [DBOS Transact](https://github.com/dbos-inc/dbos-ts).
+In this tutorial, we'll modify the example application from our [quickstart](./quickstart.md) to reliably send greetings to your friends.
```

Comment:
I still think it's valuable to add a brief overview of what concepts we'll cover here to clarify what our workflows look like. For example, "We'll show you how to create HTTP endpoints and compose reliable workflows that interact with a database and make third-party API calls."